versions/3.0.md

```diff
@@ -2323,7 +2327,9 @@ While composition offers model extensibility, it does not imply a hierarchy betw
 To support polymorphism, OpenAPI Specification adds the support of the `discriminator` field.
 When used, the `discriminator` will be the name of the property used to decide which schema definition is used to validate the structure of the model.
 As such, the `discriminator` field MUST be a required field.
-The value of the chosen property has to be the friendly name given to the model under the `definitions` property.
+There are are two ways to define the value of a discriminator for an inheriting instance.
```

Comment:
Actually, I _might_ get this.  Are you saying that the value of the `discriminator` field must match the name of a definition?  Does this mean you can **only** use values that match keys of `#/components/definitions`?